google-common/src/main/scala/org/apache/pekko/stream/connectors/google/auth/ComputeEngineCredentials.scala

```diff
@@ -25,18 +25,19 @@ import scala.concurrent.Future
 @InternalApi
 private[auth] object ComputeEngineCredentials {
 
-  def apply()(implicit system: ClassicActorSystemProvider): Future[Credentials] =
+  def apply(scopes: Set[String])(implicit system: ClassicActorSystemProvider): Future[Credentials] = {
```

Comment:
Add a overloaded apply to keep mima happy

Reply:
This is internal API and its `private`. MiMa is failing for other reasons (see https://github.com/apache/incubator-pekko-connectors/pull/314#issuecomment-1890890839)